### DISCUSSION TITLE
谷歌翻译突然不能用了？

### DISCUSSION THREAD

**Discussion by LeoCaoo** [YiiGuxing/TranslationPlugin#2109] [Q&A] [answered]
![2022-09-30_21-10](https://user-images.githubusercontent.com/37643156/193277051-6409ea6d-1c61-47ea-bbdd-fa7879676a34.png)
![2022-09-30_21-15](https://user-images.githubusercontent.com/37643156/193277770-0e65c03b-f359-42db-8b8d-f401e2e4a8de.png)

**Comment by LeoCaoo** [comment, marked as answer]
今天发现是谷歌翻译停止了中国大陆的服务，怎么修改成别的地区服务呀

**Reply by YiiGuxing** [reply]
目前修改不了，即使能修改也一样访问不了的。暂时只能通过梯子，或者修改host访问，详情见 #2051

**Reply by Jason01011010** [reply]
挂了梯子，为什么还是不行呢

**Reply by YiiGuxing** [reply]
> 挂了梯子，为什么还是不行呢

配置的问题，请另阅 #2051
